### PR TITLE
Implement string labels in annotation

### PR DESCRIPTION
**Pull request by isaiahah**

### CODE REVIEW

**Comment by EricR86** [comment]
To be clear this is a new PR with the same changes proposed in PR #168 correct?

**Comment by isaiahah** [comment]
This is a new PR with the same changes as #168, but re-created for clarification as the previous PR is marked merged.

File: segway/_util.py [hunk 1]
```diff
@@ -197,11 +197,25 @@ def get_col_index(chromosome, trackname):
 
 
 def get_label_color(label):
-    color = SCHEME[label % NUM_COLORS]
-
+    # Use hash_label to replicate hashing across subprocesses, as the Python
```
Comment (EricR86) [inline segway/_util.py:200]: Are these changes still necessary with the `make_label_colors` function now in `output.py`? I thought there was going to be no more hashing.

File: segway/_util.py [hunk 2]
```diff
@@ -374,19 +387,23 @@ def find_segment_starts(data, output_label):
...
         len_data = len(data)
 
-    # unpack tuple, ignore rest
-    seg_diffs = absolute(diff(data))
-
     # if output_label is set to "full" or "subseg"
     if output_label != "seg":
+        # Equivalent to diff(data) != 0, so True where labels change
+        # Applied to both rows (seg and subseg) separately
+        seg_diffs = (data[:, 1:] != data[:, :-1])
```
Comment (EricR86) [inline segway/_util.py:394]: This line is duplicated in both conditions. I would move it out and use the `if output_label != "seg"` conditional to apply the last change (the `maximum` call).

**Review by EricR86 — COMMENTED** [review]
@isaiahah there appears to be some leftover code from the BED file changes PR that needs to be addressed. Let me know when this is done and I'll pass on for review.

File: segway/_util.py (hunk 1 above)
Comment (isaiahah) [reply segway/_util.py:200]: Thanks for catching this! I thought I deleted the hash function in merging but apparently it made its way through.

File: segway/_util.py (hunk 2 above)
Comment (isaiahah) [reply segway/_util.py:394]: The first case has the expression `data[:, 1:] != data[:, :-1]` while the second case has the expression `data[1:] != data[:-1]` due to different dimensions of `data`. Is there a single expression that can apply to both cases? 
Reshaping in the first case is an option then applying the latter expression is an option, but it requires reshaping back so I don't think it's significantly simplifying.

Comment (EricR86) [reply segway/_util.py:379]: My mistake. Keep as is. This is fine.

**Review by michaelmhoffman — APPROVED** [review]
